File: samples/swig_python/minidemo.py
#! /usr/bin/env python

import opencv
from opencv import highgui

cap = highgui.cvCreateFileCapture("../c/tree.avi")
img = highgui.cvQueryFrame(cap)
print "Got frame of dimensions (", img.width, " x ", img.height, " )"

highgui.cvNamedWindow("win", highgui.CV_WINDOW_AUTOSIZE)
highgui.cvShowImage("win", img)
highgui.cvMoveWindow("win", 200, 200)
highgui.cvWaitKey(0)

File: samples/octave/minidemo.m
#! /usr/bin/env octave


cvNamedWindow("win", CV_WINDOW_AUTOSIZE);
cap = cvCreateFileCapture("/home/x/work/sneaker/dvgrab-001.avi");
img = cvQueryFrame(cap);

printf("Got frame of dimensions (%i x %i)",img.width,img.height);

cvShowImage("win", img);
cvMoveWindow("win", 200, 200);
cvWaitKey(0);

octimg = cv2im(img);

